src/components/ServerVersion.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from "react";
+import { Icon, Tooltip } from "@canonical/react-components";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { fetchSettings } from "api/server";
+
+const Navigation: FC = () => {
+  const { data: settings } = useQuery({
+    queryKey: [queryKeys.settings],
+    queryFn: fetchSettings,
+  });
+
+  const serverVersion = settings?.environment?.server_version;
+  const majorVersion = serverVersion?.includes(".")
+    ? serverVersion.split(".")[0]
+    : undefined;
+  const isOldVersion = majorVersion ? parseInt(majorVersion) < 5 : false;
```

Comment:
Maybe this hard-coded value (5) here should be moved to a const, so that we can find it and update it more easily in the future? This is likely to be changed with time, no?

Reply:
right, good idea!